weasyprint/css/computed_values.py

```diff
@@ -218,37 +249,21 @@ def compute(element, pseudo_type, specified, computed, parent_style,
         function = getter(name)
         already_computed_value = False
 
-        if value and isinstance(value, tuple) and value[0] == 'var()':
-            variable_name, default = value[1]
-            computed_value = _resolve_var(computed, variable_name, default)
-            if computed_value is None:
-                new_value = None
-            else:
-                prop = PROPERTIES[name.replace('_', '-')]
-                if prop.wants_base_url:
-                    new_value = prop(computed_value, base_url)
-                else:
-                    new_value = prop(computed_value)
-
-            # See https://drafts.csswg.org/css-variables/#invalid-variables
-            if new_value is None:
-                try:
-                    computed_value = ''.join(
-                        token.serialize() for token in computed_value)
-                except BaseException:
-                    pass
-                LOGGER.warning(
-                    'Unsupported computed value `%s` set in variable `%s` '
-                    'for property `%s`.', computed_value,
-                    variable_name.replace('_', '-'), name.replace('_', '-'))
-                if name in INHERITED and parent_style:
-                    already_computed_value = True
-                    value = parent_style[name]
-                else:
-                    already_computed_value = name not in INITIAL_NOT_COMPUTED
-                    value = INITIAL_VALUES[name]
-            else:
-                value = new_value
+        if value and isinstance(value, list):
+            for i, val in enumerate(value):
+                if not isinstance(val, tuple) or val[0] != 'var()':
+                    continue
+                new_value, already_computed_value = \
+                    resolve_var(val, name, computed)
+                if name == 'content':
```

Comment:
Hmmmm… There’s definitely something wrong with this. `content` shouldn’t be different from other properties. I’m not sure to understand how this PR really works, but having these special cases doesn’t look like a good solution for me :unamused:.